### PR TITLE
Fix getData image load check

### PR DESCRIPTION
**Pull request by YevheniiKravchenko**

### CODE REVIEW

File: exif.js [hunk 1]
```diff
@@ -928,10 +928,10 @@
     }
 
     EXIF.getData = function(img, callback) {
-        if ((self.Image && img instanceof self.Image)
-            || (self.HTMLImageElement && img instanceof self.HTMLImageElement)
-            && !img.complete)
-            return false;
+        const isImage = (self.Image && img instanceof self.Image)
```
Comment (diegocr) [inline exif.js]: why `const`? you'll be breaking a few browsers using that.

**Comment by diegocr** [comment]
#101 seems a better approach to fix the same issue :)

File: exif.js (hunk 1 above)
Comment (YevheniiKravchenko) [inline exif.js]: `const` is now standard and best practice for variables declaration. And it's widely supported even without babel. http://caniuse.com/#search=const

**Comment by YevheniiKravchenko** [comment]
Closing this PR. Anyway it was fix for forked repo, which I spontaneously posted as a PR to your repo. 
But in general I suggest to break so long conditions to smaller ones to make code more readable

**Pull request closed without merging**